tests/upload_test.py

```diff
@@ -0,0 +1,48 @@
+import pandas as pd
+
+from depmapomics.config import *
+from depmapomics import tracker as track
+from depmapomics import upload
+
+
+def test_gumbo_formatting():
+    # ensures the tables in gsheet are formatted correctly
+    trackerobj = track.initTracker()
+
+    pr_table = trackerobj.read_pr_table()
+    assert [
+        "ModelCondition",
+        "CDSID",
+        "PublicReleaseDate",
+        "ConsortiumReleaseDate",
+        "InternalReleaseDate",
+        "IBMReleaseDate",
+    ] in pr_table.columns, "missing column(s) in omicsProfile table"
```

Comment:
this should be in a config and given as params

Reply:
What do you think about using pandera to validate these columns are present? (and putting that validation into read_pr_table() so that we have a guarentee on the shape of pr_table)